docs/digital_forensic_research_workshop.md

```diff
@@ -1,56 +1,115 @@
+
+The DFRWS addresses three major objectives:
 
 * Define the need and create the processes for the incorporation of a
-  rigorous scientific method as a fundamental tenant of the evolving
-  discipline of Digital Forensic Science.
-* Develop a research agenda that considers practitioner requirements,
-  multiple investigative environments and emphasizes real world
-  usability.
-* The discovery, explanation and presentation of conclusive, persuasive
-  evidence that will meet the heightened scrutiny of the courts and
-  other decision-makers in military and civilian environments.
+  rigorous scientific method as a fundamental part of the evolving
+  discipline of Digital Forensics.
+* Advance Digital Forensics research and development that considers 
+  practitioner requirements, multiple investigative environments, and 
+  emphasizes real world usability.
+* The discovery, explanation and presentation of reliable, persuasive
+  digital evidence that will meet the heightened scrutiny of the courts 
+  and other decision-makers in civilian and military contexts.
 
+## International Community
 Since 2001, an annual workshop has been held in various cities around
-the US to present and discuss research topics. Archives are available as
-well as a more detailed history.
+North America, and the Jubilee 25th DFRWS conference will be held in 
+Chicago. To meet the needs of the growing international Digital Forensics 
+community, in 2014 the DFRWS started holding annual conferences in Europe,
+and in 2021 started holding annual conferences in APAC.
+
+All papers are peer reviewed and published open access with presentation 
+materials in the DFRWS Papers & Presentations online catalog. Video 
+recordings of selected presentations are periodically posted on the 
+DFRWS YouTube channel.
+
+## Applied Research
+In 2019, the DFRWS created DFIR Review to provide practitioners with a
+venue for community-reviewed applied research and testing in Digital 
+Forensics & Incident Response (DFIR). Articles from DFIR Review have 
+been cited in multiple court proceedings.
+
+## DFRWS Supported Initiatives
+DFRWS provides infrastructure sponsorship in collaboration with the 
+Linux Foundation for the Cyber Domain Ontology (CDO) community project. 
+One focus of the CDO project is Cyber-investigation Analysis Standard 
+Expression (CASE) which was formally launched at DFRWS EU 2015 and 
+presented as “Leveraging CybOX to Standardize Representation and 
+Exchange of Digital Forensic Information.”
 
-## Sponsorships
+Another DFRWS supported project called Systematic Objective-based 
+Listing of Various Established [Digital] Investigation Techniques 
+(SOLVE-IT) to establish a resource and tools for systematic error 
+mitigation for digital evidence. This initiative was started by the 
+community at DFRWS US 2023, and formally launched at DFRWS EU 2025 
+and presented as “SOLVE-IT: A proposed digital forensic knowledge 
+base inspired by MITRE ATT&CK.”
```

Comment:
Note that there are specific branding terms here https://attack.mitre.org/resources/legal-and-branding/